crates/release_plz_core/src/release.rs

```diff
@@ -59,7 +68,7 @@ pub async fn release(input: &ReleaseRequest) -> anyhow::Result<()> {
                 info!("{} {}: already published", package.name, package.version);
                 return Ok(());
             }
-            release_package(&mut index, package, input)?;
+            release_package(&mut index, package, input).await.unwrap();
```

Comment:
```suggestion
            release_package(&mut index, package, input).await?;
```
We can return an error here instead of panicking. Can you also do the same in the rest of your code?
You can replace the `unwrap`s with `?` and return `anyhow::Result` from the function.